Sources/Accord.Statistics/TimeSeries/ACF.cs

```diff
@@ -0,0 +1,87 @@
+// Accord Statistics Library
+// The Accord.NET Framework
+// http://accord-framework.net
+//
+// Copyright © César Souza, 2009-2017

```

Comment:
Are the functions based on another code (maybe the Python library we discussed before?) If yes, then please include here the license headers of the original code as well. Also please feel free to add your name to the copyright headers if you wish. I will ask about your permission to share the code under the appropriate license at end of the PR

Reply:
It partially is based on the Python ACF function but this apporach is widely used. I will addin the Python reference. After I make changes, should I just commit with modified code?

Reply:
Hi @xieliaing, great! Don't worry, it is perfectly fine to base code on other existing implementations, but we must absolutely get the licensing terms right. Yes, you can commit directly to this branch, and the pull request will be updated.

Reply:
added my name to copyright header. 
I also reviewed the acf function in Python StatsModels, my implementation is different in the sense that I directly implemented the method using Fourier Transformation while ACF function in StatsModel leverages its ACOVF function which has an option to use FFT based method. I think given this fact, we are no longer considered relying on StatsModel implementation.